include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf_coupled_solver.cpp

```diff
@@ -225,22 +226,48 @@ TimeIntBDFCoupled<dim, Number>::do_timestep_solve()
     ((this->time_step_number - 1) % this->param.update_preconditioner_coupled_every_time_steps ==
      0);
 
-  if(this->param.nonlinear_problem_has_to_be_solved())
+  // calculate Sum_i (alpha_i/dt * u_i) and store
```

Comment:
pulled out the `sum_alpha_ui`

Reply:
Was this part also copied before or why does this change appear now?